commands.js

```diff
@@ -1701,6 +1706,7 @@ var commands = exports.commands = {
 			});
 		});
 	},
+	updateserverhelp: ["/updateserver - updates the server to a newer version. Requires: Console Access"],
```

Comment:
'updateserverhelp: ["/updateserver - updates the server to a newer version. Requires: Console Access"],' Capital letter is missing on "Updates".